src/CoreBundle/Tests/CronJob/fixtures/CronJobThatExtendsTCMSCronJob.php

```diff
@@ -6,8 +6,10 @@
 
 class CronJobThatExtendsTCMSCronJob extends TCMSCronJob
```

Comment:
This is what must be changed: not extending TCMSCronJob

Reply:
resolved.